Chess/include/ConstUtils.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+/**
+* @file ConstUtils.h
+* this is simply to hold all the constants used in the project and a few utility functions that
+* are used in multiple files so including them in a class would be redundant or cause circular dependencies
+*/
+static const char WHITE = 'w';
+static const char BLACK = 'b';
+static const char WHITE_KING = 'K';
+static const char BLACK_KING = 'k';
+static const char WHITE_ROOK = 'R';
+static const char BLACK_ROOK = 'r';
+static const char WHITE_BISHOP = 'B';
+static const char BLACK_BISHOP = 'b';
+static const char WHITE_KNIGHT = 'N';
+static const char BLACK_KNIGHT = 'n';
+static const char WHITE_QUEEN = 'Q';
+static const char BLACK_QUEEN = 'q';
+static const char WHITE_PAWN = 'P';
+static const char BLACK_PAWN = 'p';
+static const char EMPTY_SPACE = '#';
```

Comment:
Can you think of a way to know if it's a capital case or a lowercase? If so, can you think of a way to reduce the number of consts by half?